src/marilib.src

```diff
@@ -1,59 +1,63 @@
-// 
```

Comment:
Why removing all of the // at the top of the comments? I'll change it back anyway xd It's sort of a habit commenting in MS